### PR TITLE
Make scripts/ci/libraries Google Shell Guide Compliant

### PR DESCRIPTION
**Pull request by potix2**
Part of #10576

I checked the following scripts but they doesn't have anything we need to change.

*  ./scripts/ci/libraries/_permissions.sh
*  ./scripts/ci/libraries/_script_init.sh
*  ./scripts/ci/libraries/_runs.sh
*  ./scripts/ci/libraries/_local_mounts.sh
*  ./scripts/ci/libraries/_all_libs.sh

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Comment by potix2** [comment]
@potiuk I have completed the task. Could you review this changes?

**Comment by potiuk** [comment]
Sorry for the little delay. I will do it today :)

File: scripts/ci/libraries/_build_images.sh [hunk 1]
```diff
@@ -959,41 +962,41 @@ function build_images::build_prod_images() {
 #  $3, $4, ... - target tags to tag the image with
 function build_images::wait_for_image_tag() {
 
-    IMAGE_NAME="${1}"
-    IMAGE_SUFFIX=${2}
+    local image_name="${1}"
+    local image_suffix=${2}
```
Comment (potiuk) [inline scripts/ci/libraries/_build_images.sh]:
```suggestion
    local image_suffix="${2}"
```

**Review by potiuk — APPROVED** [review]
LGTM. I want to do some last verification it it works for a few cases and there is one small quote addition but it looks nice!

**Comment by potiuk** [comment]
Thanks!

**Pull request merged**